src/README.md

```diff
@@ -0,0 +1,94 @@
+# OddSite
```

Comment:
@SondraE did you mean to duplicate the README here?

Reply:
Nope.